libraries/Microsoft.BotKit.Adapters.Slack/Microsoft.BotKit.Adapters.Slack.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>botbuilder_slack_adapter</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.WebApi" Version="5.2.7" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.4.3" />
```

Comment:
As this is in our main solution, it should follow the same project scheme our other .Net Core projects follow. IT should apply all the same rules, and versioning standards.

Reply:
We updated the projects. Thanks!